suites/create.js

```diff
@@ -309,6 +309,40 @@ export function runDataIntegrityProofFormatTests({
         }
       });
     }
+    it('If an @context property is not provided in a document that is ' +
+    'being secured or verified, or the Data Integrity terms used in ' +
+    'the document are not mapped by existing values in the @context ' +
+    'property, implementations MUST inject or add an @context property ' +
+    'with a value of https://w3id.org/security/data-integrity/v2.',
+    async function() {
+      if(!issuer) {
+        throw new Error(`Expected ${vendorName} to have an issuer.`);
+      }
+      const vc = structuredClone(validVc);
+      const expectedContext = 'https://w3id.org/security/data-integrity/v2';
+      // remove the vc's context and expect context injection to occur
+      delete vc['@context'];
```

Comment:
Since we already test for missing context, wouldn't the implementation be expected to reject this payload if it has no or an empty context? This seems to contradict [this test](https://github.com/w3c/vc-data-model-2.0-test-suite/blob/445f5f29af893e0b675442273abdf1b053cf12db/tests/10-vcdm2.js#L113)?

Reply:
This test seems to only be valid outside of Verifiable Credentials, for other types of documents being secured with DI.

Reply:
@PatStLouis in the spec there is nothing that says `@context` is required at this level (there is in the Vc 2.0 Data Model). The existing test in D.I. that used to check for the expected contexts is now only run when this suite is running on non-DataIntegrityProofs unless I am missing a test somewhere?

Reply:
if I tag my implementation:
`["vc2.0", "eddsa-rdfc-2022"]`

I will get 1 test making sure I reject empty context (the one I linked) then 1 test making sure I accept empty context and inject a url (this test). I can't pass both of these tests with the same implementation. This is what I'm trying to highlight here. It's true that the Data Integrity doesn't make any requirements as to the document to be signed, however the test suites will run against implementations which are expected to be conformant with the VCDM.

Maybe I'm the one missing something

Reply:
> The existing test in D.I. that used to check for the expected contexts is now only run when this suite is running on non-DataIntegrityProofs

Could you explain this more I'm not sure I fully understand it

Reply:
yeah I have trouble parsing that statement to, anyways 

https://github.com/w3c-ccg/data-integrity-test-suite-assertion/blob/001e58a4c0878b2dae3fbdb68b2d55d67c5de612/suites/create.js#L81-L120


Also, WRT to the VC 2.0 test and testing for the presence of `@context` and then relevant contexts, that could be a verifier test as we can't be sure if an issuer will just adds the contexts.